engine/runtime/src/test/java/org/enso/interpreter/test/EqualsTest.java

```diff
@@ -55,11 +55,7 @@ public static void disposeContext() {
   private static Object[] fetchAllUnwrappedValues() {
     var valGenerator =
         ValuesGenerator.create(
-            context,
-            ValuesGenerator.Language.ENSO,
-            ValuesGenerator.Language.JAVA,
-            ValuesGenerator.Language.JAVASCRIPT,
-            ValuesGenerator.Language.PYTHON);
+            context, ValuesGenerator.Language.ENSO, ValuesGenerator.Language.JAVA);
```

Comment:
I know that `Language.PYTHON` has never been supported in `ValuesGenerator` so far, but does this mean that we lose `Language.JAVASCRIPT` values in `EqualsTest`? Did you make this change to make the test run faster?

Reply:
`engine/runtime` tests do not support polyglot languages like Python or JavaScript. They never did.

Now, when I added one JavaScript value into `maps()` I need to disable `Language.JAVA_SCRIPT` when running the `engine/runtime` tests.

If you want to test JavaScript, Python, etc. you need to create `EqualsPolyglotTest`, just like I did in this PR for existing `Meta*Test`s.

Reply:
> engine/runtime tests do not support polyglot languages like Python or JavaScript. They never did.

Oh right, I forgot about that one. Now, we have `runtime-with-polyglot` for that.

> If you want to test JavaScript, Python, etc. you need to create EqualsPolyglotTest, just like I did in this PR for existing Meta*Tests.

We definitely should do that. Tracked in https://github.com/enso-org/enso/issues/6826